openapi.yml

```diff
@@ -715,6 +715,411 @@ paths:
+                        format: uint64
+                        description: The number of critical alerts
+        "400":
+          description: Malformed request
+          content:
+            text/plain:
+              schema:
+                type: string
+              examples:
+                invalidLimit:
+                  summary: Invalid limit example
+                  value: "limit must be greater than or equal to -1"
+                invalidOffset:
+                  summary: Invalid offset example
+                  value: "offset must be greater than or equal to 0"
+        "500":
+          description: Internal server error
+          content:
+            text/plain:
+              schema:
+                type: string
+  /bus/alerts/dismiss:
+    post:
+      summary: Dismiss alerts
+      description: Dismisses the specified alerts.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: "#/components/schemas/Hash256"
+      responses:
+        "200":
+          description: Successfully dismissed alerts
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: Internal server error
+          content:
+            text/plain:
+              schema:
+                type: string
+  /bus/alerts/register:
+    post:
+      summary: Register an alert
+      description: Registers a new alert.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                severity:
+                  $ref: "#/components/schemas/Alert"
+      responses:
+        "200":
+          description: Successfully registered alert
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: Internal server error
+          content:
+            text/plain:
+              schema:
+                type: string
+
+  /bus/buckets:
+    get:
+      summary: Get all buckets
+      description: Returns all known buckets.
+      responses:
+        "200":
+          description: Successfully retrieved buckets
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Bucket"
+        "500":
+          description: Internal server error
+          content:
+            text/plain:
+              schema:
+                type: string
+    post:
+      summary: Create bucket
+      description: Create a new bucket.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                  pattern: (?!(^xn--|.+-s3alias$))^[a-z0-9][a-z0-9-]{1,61}[a-z0-9]$
```

Comment:
You can replace all your bucket names with references to the bucket type that already contains this pattern.